hearts/hearts.py

```diff
@@ -134,6 +139,24 @@ def set_turn_counter(self):
             if Card('2', 'c') in self.hands[self.players[index]]:
                 self.turn_counter = index
 
+    def pass_cards(self, card_selections):     # {player:[Card]} --> None
```

Comment:
`card_selections` is user input, so we need to be more careful. E.g., a nefarious user (or a bug in our code) might result in `card_selections` containing cards that are not in the player's hand.

I think we should assume for now that the input to `pass_cards` is already validated (so you can remove the `ValueError` and the if/else), and write a second method on `Round` called `is_valid_pass_for_player(self, player, cards)`. It should return true if and only there are three chosen cards and they're all in the player's hand.